pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>hal-explorer</artifactId>
-    <version>0.15.1-SNAPSHOT</version>
+    <version>0.16.0</version>
     <name>HAL Explorer</name>
     <description>WebJar for HAL Explorer</description>
     <url>http://webjars.org</url>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <upstream.url>https://github.com/toedter/hal-explorer/releases/download/v015.0/hal-explorer-${version.unsnapshot}.zip</upstream.url>
+        <upstream.url>https://github.com/toedter/hal-explorer/releases/download/v${project.version}/hal-explorer-${project.version}.zip</upstream.url>
```

Comment:
```suggestion
        <upstream.url>https://github.com/toedter/hal-explorer/releases/download/v${version.unsnapshot}/hal-explorer-${version.unsnapshot}.zip</upstream.url>
```

Reply:
Using `version.unsnapshot` makes it so we don't have to change anything when the version changes from `0.16.0-SNAPSHOT` to `0.16.0`.